BrokenTC2/TC/GearHandler.cpp

```diff
@@ -168,4 +169,27 @@ void GearHandler::gearDown() {
     switchSeqGear(false);
 }
 
+void GearHandler::holdFirstGear() {
+  if (m_settings.holdFirstGearWithClutch) {
+    windows::sendKeyboardEvent(m_settings.clutch,true);
+    std::this_thread::sleep_for(std::chrono::milliseconds(m_settings.keyDownTime));
+    windows::sendKeyboardEvent(m_settings.g1,true);
+  } else {
+    windows::sendKeyboardEvent(m_settings.g1,true);

```

Comment:
As windows::sendKeyboardEvent(m_settings.g1,true); is common to both the "if" and "else", you could just put it outside and not adding an "else"
This would be read the code as "if the cluthc is needed, press it before pressing the gear key" ^^